scripts/generate_sheet.py

```diff
@@ -35,8 +35,7 @@ def generate_sheet(model_name, token, days, threshold):
         p = model.classify(bug, probabilities=True)
         probability = p[0]
         if len(probability) > 2:
-            index = np.argmax(probability)
-            prediction = model.class_names[index]
+            prediction = model.le.inverse_transform([np.argmax(probability)])[0]
```

Comment:
Thanks for this fix, could you move it to a separate PR though, as it's a separate fix?